internal/commands/inits/data/status_shortcuts.sh

```diff
@@ -25,7 +25,7 @@ scmpuff_status() {
   local e=1
   for file in $files; do
     export $scmpuff_env_char$e="$file"
-    let e++
+    (( e++ ))
```

Comment:
Is this syntax fully POSIX compatible?

Reply:
This was automatically flagged by the shell linter you have in the repo, can’t see the exact message any more but I believe it mentioned that this has been supported in Bash for decades